### ISSUE TITLE
False Positive - Unexpected spacing after "("

### ISSUE THREAD

**Issue by JayNewstrom** [shyiko/ktlint#245]
ktlint is triggering a false positive when long constructor line is broken into two.

```kotlin
class ClassWithNoConstructorArguments @Inject constructor(
) : InterfaceOne, InterfaceTwo, InterfaceThree, InterfaceFour, InterfaceFive, InterfaceSix
```

I need to break the line into two so that the max line length isn't exceeded, and it seems best to break at the constructor, incase I need to add dependencies later.

Is this an expected error, is is this valid when the line is long?

**Comment by shyiko** [comment]
How about a newline after ":" instead?   
(https://kotlinlang.org/docs/reference/coding-conventions.html#class-header-formatting)

e.g. 
```kotlin
class ClassWithNoConstructorArguments @Throws constructor() :
    MyLongHolder<MyFavouriteVeryLongClass>(),
    SomeOtherInterface,
    AndAnotherOne
```

**Comment by JayNewstrom** [comment]
That works for me. Thanks.
